File: src/016. Power digit sum/016.go
package main

import (
	"fmt"
	"math/big"
)

func main() {
	var power big.Int
	power.Exp(big.NewInt(2), big.NewInt(1000), nil)
	sumDigits := 0
	for _, char := range(power.String()) {
		digit := char - '0'
		sumDigits += int(digit)
	}
	fmt.Println(sumDigits)
}

File: src/016. Power digit sum/016.java
import java.math.*;

public class Main {
    public static void main(String[] args) {
        BigInteger power = BigInteger.valueOf(2).pow(1000);
        byte[] arr = power.toString().getBytes();
        int sum = 0;
        for (byte digit : arr) {
            sum += digit - '0';
        }
        System.out.println(sum);
    }
}
